src/main/resources/regExs.json

```diff
@@ -0,0 +1,4 @@
+{
+  "invalidName" : "(\\W)",
+  "validEmail" : "^(([^<>()\\[\\]\\\\.,;:\\s@\"]+(\\.[^<>()\\[\\]\\\\.,;:\\s@\"]+)*)|(\".+\"))@((\\[[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}])|(([a-zA-Z\\-0-9]+\\.)+[a-zA-Z]{2,}))$"
```

Comment:
Key value should have same name as in subscription json objects field names.

Reply:
Noticed this is not in EI code,,, then its OK with those names. InvalidString is maybe a better name for InvalidName,, since the regex is not specific for just name fields.